### DISCUSSION TITLE
The Package ID reserved. You can upload your package with a different package id.

### DISCUSSION THREAD

**Discussion by jiten87** [NuGet/Home#11237] [Q&A]
I am trying to upload a NuGet package and getting the error "The Package ID reserved. You can upload your package with a different package id.".
Also when searching the same getting no package.

**Comment by erdembayar** [comment]
Please check https://github.com/NuGet/Home/discussions/11104#discussioncomment-1121592
Short answer: Even you don't find any package still prefix can be already reserved.
